service-commands/scripts/A

```diff
@@ -190,14 +205,24 @@ def setup(name, azure, service, config, user):
 
     ip = proc.stdout.decode().strip()
     if proc.returncode != 0:
-        input(f"Creating new instance '{name}'. Press Enter to continue...")
+        input(f"Creating new instance '{name}' from prebaked image: {fast}. Press Enter to continue...")
```

Comment:
this log should only print the `prebaked image` part if the fast flag was true? i think this prints in all cases

Reply:
Yeah, I'll make that change!

Reply:
oops. i shold go back and make this change orz